runbotics/runbotics-orchestrator-ui/src/components/ResizableDrawer/ResizableDrawer.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Drawer } from '@mui/material';
+import { DrawerProps } from '@mui/material/Drawer/Drawer';
+import useWrappedState from 'src/hooks/useWrappedState';
+
+const INITIAL_WIDTH = 300;
+const MIN_WIDTH = 250;
+const MAX_WIDTH = 600;
+
+const Dragger = styled('div')(({ theme }) => ({
+    width: '3px',
```

Comment:
I suggest you keep the order of your styles, it will be easier to maintain

(1) positioning (2) structure (3) size (4) colors, transitions (5) font (6) rest